fftvis/beams.py

```diff
@@ -101,3 +101,98 @@ def get_apparent_flux_polarized(beam: np.ndarray, flux: np.ndarray):  # pragma:
         beam[0, 1, isrc] = i01 * flux[isrc]
         beam[1, 0, isrc] = np.conj(i01) * flux[isrc]
         beam[1, 1, isrc] = i11 * flux[isrc]
+
+@nb.jit(
+    nopython=True,
+    parallel=False,
+    nogil=False
+)
+def get_apparent_flux_polarized(beam: np.ndarray, coherency: np.ndarray): # pragma: no cover
```

Comment:
Is there a way to do this using matrix products? I'm on my phone right now so I can't think through the math properly, but I feel like there must be a way.

Reply:
Do you mean through a np.dot or np.einsum call?

Reply:
Yeah. Though I see you're using `nb.jit` so maybe that's a better approach to threading?

Reply:
I see. I can definitely do it that way, but we had previously implemented this kind of function to save memory when computing the beam/sky product. If we do go back to np.einsum, we should probably be consistent across the different cases

Reply:
OK, perhaps I just can't remember what we had been doing. If `numba` is already a dependency, then this seems fine.